lib/tapjoy/ldap/key/add.rb

```diff
@@ -0,0 +1,51 @@
+module Tapjoy
```

Comment:
not sure why highlighting isn't turned on for this file, but it looks weird. No obvious issues with code though.